src/walkthrough.md

```diff
@@ -186,10 +186,17 @@ request that reviewer by leaving a comment on the thread with `r?
 @reviewer-github-id` (e.g. `r? @eddyb`). If you don't know who to request,
 don't request anyone; the bot will assign someone automatically based on which files you changed.
 
-The reviewer may request changes before they approve your PR. Feel free to ask
-questions or discuss things you don't understand or disagree with. However,
+The reviewer may request changes before they approve your PR, they may mark the PR with label 
+"S-waiting-on-author" after leaving comments, this means that the PR is blocked on you to making 
+some requested changes. When you finished iterating on the changes, you can mark the PR as 
+`S-waiting-on-review` again by leaving a comment with `@rustbot ready`, this will remove the 
+`S-waiting-on-author` label and add the `S-waiting-on-review` label.
+
+Feel free to ask questions or discuss things you don't understand or disagree with. However,
 recognize that the PR won't be merged unless someone on the Rust team approves
-it.
+it. If a reviewer leave a comment like `r=me after fixing ...`, that means they approve the PR and 
+you can merge it with comment with `@bors r=reviwer-github-id`(e.g. `@bors r=eddyb`) to merge it 
+after fixing lefted trivial issues.
```

Comment:
Worth mentioning that r=someone requires permission and the delegate cmd gives it to you.